mod/projects/set/type_plus_right/project/metric.rb

```diff
@@ -42,17 +42,15 @@ def filter_card
   end
 
   view :core do
-    wrap_with :div, class: "progress-bar-table" do
-      metric_progress_table
-    end
+    nest Card.fetch(card.name, :project), view: :content, items: { view: :bar }
```

Comment:
The point of all these bar-related changes is that by moving these into a separate card, we can automatically get paging, which is badly needed on these project pages.